.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 .project
+.idea
```

Comment:
Unnecessary change, should be in your global gitignore file IMO

Reply:
can we just let it here for others contributors?

Reply:
Well since this is specific to a certain IDE I say no, because everyone can decide whatever IDE they use for js development. Was this an Android or iOS project I would agree in adding it to the projects gitignore file, because the options there are limited.

Reply:
See also; https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000154070-How-to-gitignore-idea-files?page=1#community_comment_115000258150

Reply:
removed